src/site/content/en/third_party/chrometober-2022/hoisted.690feb9c.js

```diff
@@ -0,0 +1,2 @@
+const De=()=>{const h=document.querySelectorAll("[data-scroll-trigger]"),p=document.querySelectorAll(".page__side"),b=w=>{for(let N of p)N.classList.remove("in-view");let k=w.reverse().find(N=>N.isIntersecting),I;if(!k&&w.length>1)return;!k&&w.length===1?(I=(!w[0].isIntersecting&&w[0]).target.dataset.scrollTrigger-1,k=document.querySelector(`[data-scroll-target="${I}"]`)):(k=k.target,I=parseInt(k.dataset.scrollTrigger));let W=document.querySelector(`[data-scroll-target="${I}"] .page__side--back`),E=document.querySelector(`[data-scroll-target="${I+1}"] .page__side--front`);p.forEach(N=>N.setAttribute("inert","")),W?.removeAttribute("inert"),W?.classList.add("in-view"),E?.removeAttribute("inert"),E?.classList.add("in-view"),window.ga&&window.ga("send","event",{event_action:"Section Order",event_label:`pages-${I}-and-${I+1}`,event_category:"Chrometober Book",non_interaction:!0})};(w=>{const k=new IntersectionObserver(b,{threshold:.9});for(let I of w)k.observe(I)})(h)};De();(function(){function h(i,n){for(var e=0;e<n.length;e++){var t=n[e];t.enumerable=t.enumerable||!1,t.configurable=!0,"value"in t&&(t.writable=!0),Object.defineProperty(i,t.key,t)}}function p(i,n,e){return n&&h(i.prototype,n),e&&h(i,e),Object.defineProperty(i,"prototype",{writable:!1}),i}function b(){return b=Object.assign||function(i){for(var n=1;n<arguments.length;n++){var e=arguments[n];for(var t in e)Object.prototype.hasOwnProperty.call(e,t)&&(i[t]=e[t])}return i},b.apply(this,arguments)}function T(i,n){i.prototype=Object.create(n.prototype),i.prototype.constructor=i,w(i,n)}function w(i,n){return w=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e},w(i,n)}function k(i){if(i===void 0)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return i}function I(i,n){(n==null||n>i.length)&&(n=i.length);for(var e=0,t=new Array(n);e<n;e++)t[e]=i[e];return t}function W(i,n){var e=typeof Symbol<"u"&&i[Symbol.iterator]||i["@@iterator"];if(e)return(e=e.call(i)).next.bind(e);if(Array.isArray(i)||(e=function(r,o){if(r){if(typeof r=="string")return I(r,o);var a=Object.prototype.toString.call(r).slice(8,-1);return a==="Object"&&r.constructor&&(a=r.constructor.name),a==="Map"||a==="Set"?Array.from(r):a==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(a)?I(r,o):void 0}}(i))||n&&i&&typeof i.length=="number"){e&&(i=e);var t=0;return function(){return t>=i.length?{done:!0}:{done:!1,value:i[t++]}}}throw new TypeError(`Invalid attempt to iterate non-iterable instance.
```

Comment:
I think we need a license of this file. We can probably put a license for whole dir in third_party/chrometober-2022/LICENSE

Reply:
Added 👍